### PR TITLE
Add `netdata_healthcheck_target` configuration

### PR DESCRIPTION
**Pull request by plasticrake**
Add configuration `netdata_healthcheck_target` that is passed to `NETDATA_HEALTHCHECK_TARGET` environment variable for netdata docker image to control healthcheck: https://learn.netdata.cloud/docs/installing/docker#health-checks

### CODE REVIEW

**Comment by felipecrs** [comment]
Thanks for the feature, @plasticrake. I really wonder, are you going to use it?

File: netdata/DOCS.md [hunk 1]
```diff
@@ -75,6 +75,12 @@ Refer to
 
 Note that `lm-sensors` comes preinstalled already.
 
+### Option: `netdata_healthcheck_target`
+
+Change or disable the default docker health check from Netdata.
```
Comment (felipecrs) [inline netdata/DOCS.md:80]:
```suggestion
Allows to control how the docker health checks from Netdata run. For example: `cli`.
```

**Comment by felipecrs** [comment]
Thanks for your contribution!

**Comment by plasticrake** [comment]
> Thanks for the feature, @plasticrake. I really wonder, why are you going to use it?

Yup, right now my container shows unhealthy because It's configured to only send data to another netdata instance and the API call the default health check uses isn't there.

**Comment by felipecrs** [comment]
Got it. Thanks.

**Pull request merged**